Example/Emission.xcodeproj/project.pbxproj

```diff
@@ -615,6 +709,94 @@
 			};
 			name = Release;
 		};
+		51D5E9101D92814800C74594 /* Deploy */ = {
```

Comment:
Aye aye, this configuration adds a CPP macro called `DEPLOY` which we can then use in the code to conditionally compile in the AppHub stuff.